app/models/manageiq/providers/google/cloud_manager/event_parser.rb

```diff
@@ -0,0 +1,18 @@
+module ManageIQ::Providers::Google::CloudManager::EventParser
+  def self.event_to_hash(event, ems_id)
+    log_header = "ems_id: [#{ems_id}] " unless ems_id.nil?
+
+		_log.debug { "#{log_header}event: [#{event["event_type"]}]" }
```

Comment:
I grabbed the sample log line from another provider, the hash key should actually be `"eventType"` based on the `event_hash` below.  Also use spaces not tabs here.